bedrock/products/templatetags/misc.py

```diff
@@ -148,7 +149,12 @@ def vpn_subscribe_link(
     selected_plan = available_plans.get(plan, VPN_12_MONTH_PLAN)
     plan_id = selected_plan.get("id")
 
-    product_url = f"{settings.VPN_SUBSCRIPTION_URL}subscriptions/products/{product_id}?plan={plan_id}"
+    if switch("vpn-subplat-next"):
+        product_slug = "mozillavpnstage"
```

Comment:
Nit, non-blocking: any reason to have `product_slug` as a variable and not just in the f-string a couole of lines later?

Reply:
Left it here for when we get the prod slug